tests/scripts/mbedtls_test.py

```diff
@@ -0,0 +1,343 @@
+# Greentea host test script for on-target tests.
```

Comment:
This script is not dependent on mbed-os. Rather it is a generic host test script for test suites testing  on the target that implements Greentea interface. It very mcuh belongs in this repo as its business logic is test suites testing.
For mbed-os: When importing mbedtls to mbed-os this script is copied to mbed-os dir with generated test suites code.
It can also be used with baremetal testing as long as Greentea is used. For example Charon TLS testing.

Reply:
PyDoc description is [next](https://github.com/ARMmbed/mbedtls/pull/930/files#diff-81116503c9488b82f0e40588f805da3dR22) to this block in quotes. Here it is just summary/label of this file.